src/ServicePulse.Host/vue/src/views/EventsView.vue

```diff
@@ -0,0 +1,184 @@
+<script setup>
+import { ref, onMounted, onUnmounted } from "vue";
+import { licenseStatus } from "../composables/serviceLicense.js";
+import { useGetEventLogItems } from "../composables/serviceEventLogItems.js";
+import TimeSince from "../components/TimeSince.vue";
+import LicenseExpired from "../components/LicenseExpired.vue";
+
+const eventLogItems = ref([]);
+const eventLogItemsPerPage = ref(25);
+
+function getEventLogItems() {
+    return useGetEventLogItems().then((result) => {
```

Comment:
was discussed in the pulse and control channel that samples like this can be converted to async/await rather than callback syntax

Reply:
Converted